### ISSUE TITLE
font's not found

### ISSUE THREAD

**Issue by wendt88** [JeffreyWay/laravel-mix#328]
Hi, after the update from 0.6.0 to 0.6.3, I got this error:
```
These dependencies were not found in node_modules:

* ../fonts/MaterialIcons-Regular.eot?v=2.2.1
* ../fonts/MaterialIcons-Regular.woff2?v=2.2.1
* ../fonts/MaterialIcons-Regular.woff?v=2.2.1
* ../fonts/MaterialIcons-Regular.ttf?v=2.2.1
* ../fonts/MaterialIcons-Regular.svg?v=2.2.1
```

I use `"bootstrap-material-icons": "^2.2.0"` and include them in my sass file via `@import "node_modules/bootstrap-material-icons/scss/material-icons";`

**Comment by poldixd** [comment]
Same error. `laravel-mix@0.6.3`

```
> npm -v
4.2.0
> node -v
v6.9.4
```

webpack.mix.js
```
mix.webpackConfig({
    output: {
        publicPath: '/'
    },
    module: {
        rules: [
            {
                test: /\.css$/,
                loaders: ['style-loader', 'css-loader']
            }
        ]
    }
});

mix.copy('resources/assets/images/', 'public/images/', false)
    .copy('node_modules/font-awesome/fonts', 'public/fonts/', false)
    .copy('resources/assets/fonts', 'public/fonts/', false)
    .js('resources/assets/js/app.js', 'public/js/app.js')
    .sass('resources/assets/sass/app.scss', 'public/css')
    .sourceMaps()
    .combine([
        'node_modules/cookieconsent/build/cookieconsent.min.css'
    ], 'public/css/libs.css');
```

**Comment by kmorva** [comment]
Try with `~bootstrap-material-icons/scss/material-icons` to reference a node asset (same for Font-Awesome and Glyphicons).
Also, you don't need to do `mix.copy` for fonts and images, as these will be copied to the public directory automatically by Webpack.

**Comment by forehalo** [comment]
This error has been fixed by #312, but not tagged yet. Wait for the next release.
🔨
